dora/core/client/fs/src/main/java/alluxio/client/file/ufs/UfsBaseFileSystem.java

```diff
@@ -437,18 +438,27 @@ public void needsSync(AlluxioURI path) throws IOException, AlluxioException {
 
   @Override
   public Optional<String> submitJob(JobRequest jobRequest) {
-    throw new UnsupportedOperationException();
+    try (CloseableResource<FileSystemMasterClient> client =
```

Comment:
These methods are moved from `BaseFileSystem`. Essentially, these logic are not relevant to the `FileSystem` concept at all. They just need a place to be. Putting them here is as bad as putting them in `BaseFileSystem`. However, in order to minimize new logics in this PR, we choose to not do the refactor here.

Reply:
A `UfsBaseFileSystem` is solely backed by a UFS, and should not access the master via a `FileSystemMasterClient`. I think we should keep this method unimplemented.

Reply:
we can directly call use client at load command and delete the API here like what I did for distCP, maybe in a later PR

Reply:
I will implement that in a later PR because I think that's non-trivial code change and requires additional reviews.